docs/getting-started/integrations-cdn/next-js.md

```diff
@@ -89,18 +89,30 @@ export default CustomEditor;
 The `CustomEditor` component is ready to be used inside a page. The page's directory will differ depending on the selected routing strategy.
 
 CKEditor&nbsp;5 is a client-side text editor and relies on the browser APIs, so you need to disable server-side rendering for our custom component. You can lazily load the component using the `dynamic()` function built into Next.js.
+Prepare a wrapper for the `CustomEditor` component to load it on the client side.
```

Comment:
```suggestion

Prepare a wrapper for the `CustomEditor` component to load it on the client side.
```

Please, add this blank line as requested in https://github.com/ckeditor/ckeditor5/pull/18103#discussion_r1989220638.

Reply:
Fixed in [9b529fb](https://github.com/ckeditor/ckeditor5/pull/18103/commits/9b529fb80bd23b61a483990cbd24808038187747)